openpype/hosts/maya/plugins/publish/extract_camera_mayaScene.py

```diff
@@ -152,22 +161,29 @@ def process(self, instance):
                 with lib.suspended_refresh():
                     if bake_to_worldspace:
                         self.log.info(
-                            "Performing camera bakes: {}".format(transform))
+                            "Performing camera bakes: {}".format(transforms))
+                        attr_values = self.get_attr_values_from_data(
+                            instance.data)
+                        keep_input_connections = attr_values.get("keep_input_connections")  # noqa
                         baked = lib.bake_to_world_space(
-                            transform,
+                            transforms,
                             frame_range=[start, end],
-                            step=step
+                            step=step,
+                            copy_input_connections=keep_input_connections
                         )
-                        baked_camera_shapes = cmds.ls(baked,
-                                               type="camera",
-                                               dag=True,
-                                               shapes=True,
-                                               long=True)
-
-                        members = members + baked_camera_shapes
-                        members.remove(camera)
+                        baked_camera_shapes = set(cmds.ls(baked,
+                                                  type="camera",
+                                                  dag=True,
+                                                  shapes=True,
+                                                  long=True))
+
+                        for cam in baked_camera_shapes:
+                            self.ensure_image_plane_attachment(cam)
```

Comment:
This behavior should not happen for a `camera` instance - only for `matchmove` instance.

Reply:
Yeah, the baking should only happen for `matchmove`

Reply:
This will trigger only if `Keep input connections` is set to True (by default is False), so studio/artist can control behavior.

Reply:
The `camera` family should exclude image planes by design, and thus for that family it should not allow that option if that means it starts exporting image planes.